.travis.yml

```diff
@@ -27,6 +27,7 @@ install:
 # NOTE: mount all $GOPATH into container, then after build and test,
 # $GOPATH/bin will have expected binaries.
 script:
+  - set -e
```

Comment:
Maybe we need to run big stages independent to speed up.
How about connecting the sub stages by `&&`